src/Symfony/Component/HttpFoundation/IpUtils.php

```diff
@@ -124,6 +124,15 @@ public static function checkIp6($requestIp, $ip)
             throw new \RuntimeException('Unable to check Ipv6. Check that PHP was not compiled with option "disable-ipv6".');
         }
 
+        // Check to see if we were given a IP4 $requestIp or $ip by mistake
+        if (str_contains($requestIp, '.') || str_contains($ip, '.')) {
```

Comment:
Hi, I think it's not compliant with IP addresses like `::ffff:127.0.0.1` which is the IPv6 version of an IPv4 IP address

Reply:
see https://github.com/symfony/symfony/issues/48420 and https://github.com/symfony/symfony/issues/48421 where this has been improved. Please test that PR.